### ISSUE TITLE
fetch up-to-date limits spreadsheet

### ISSUE THREAD

**Issue by sebdalgarno** [bcgov/shinywqg#38]
there are two options:
1. get the most recent limits spreadsheet from BC Data Catalogue (using bcdata R package). This takes a few seconds and causes a loading screen in the app every time it is started up.
2. get the most recent limits spreadsheet when app is deployed. This will eliminate the loading screen but the limits sheet may be out of date if updated until the app is redeployed by Poisson.

In summary: option 1 will always be correct even if Poisson not involved but has a loading screen; option 2 requires redeployment of app by Poisson every time spreadsheet is updated, but eliminates loading screen
